packages/shared/AGENTS.md

```
@@ -1,58 +1,67 @@
-yarn test:coverage      # Tests with coverage
-yarn build              # Build (dual ESM + CJS)
-yarn ts                 # Type check
-yarn lint:fix           # Fix lint issues
+yarn test && yarn test:coverage && yarn build && yarn ts && yarn lint:fix
 ```
 
 ## Architecture
 
-### Directory Layout
-
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)